package.json

```diff
@@ -31,9 +31,12 @@
     "debug": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline -s dev --noAuth --config serverless-api.yml",
     "start-jigsaw-docs": "sls offline start -s dev --port 3015 --noAuth --config serverless-jigsaw-docs.yml",
     "auth": "serverless invoke local -s dev -f hn-single-view-api-authorizer -p authorizer/event.json --config serverless-api.yml",
+    "debug-test": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline -s test --port 3010 --noAuth",
     "unit-tests": "jest --testPathIgnorePatterns=api/test/integration --coverage",
-    "start-test": "sls offline start -s test --port 3010 --noAuth --config serverless-api.yml",
+    "start-test": "sls offline start -s test --port 3010 --noAuth --config serverless-api.yml & node ./api/test/FakeJigsaw.js",
```

Comment:
i "think" circle might need "node ./api/test/FakeJigsaw.js" to go before the rest?